sunpy/image/solar_differential_rotation.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
This should be in `sunpy.physics.differential_rotation` not `sunpy.image` image is for pixel space things only.